local_ansible_roles/solr_config/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+- name: Solr Rovio | Put solr configurations in place
```

Comment:
There was another one later too.
Other than those seems ok and can be merged (y)